### PR TITLE
use template literals for assertions

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

Module: JS1 
Week(s): 3

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: content/js1/blocks/assertions/index.md [hunk 1]
```diff
@@ -65,25 +65,21 @@ Predict and explain if the assertion will succeed or fail. Pay particular attent
...
 
 ```js
 function formatAs12HourClock() {}
 
 console.assert(
   formatAs12HourClock("08:00") === "08:00 am",
-  "current function output: %s, target output: %s",
-  formatAs12HourClock("08:00"),
-  "08:00 am"
+  `current output: ${currentOutput}, target output: ${targetOutput}`
```
Comment (illicitonion) [inline content/js1/blocks/assertions/index.md:75]: This one seems broken? It dropped the arguments, and references variables that don't exist?

We either need to extract a variable, or undo this change?

Comment (Dedekind561) [reply content/js1/blocks/assertions/index.md]: I just updated this assertion so there's no reference to any variables prior to the refactor section. Should all be in order now.

**Review by illicitonion — APPROVED** [review]
